west.yml

```diff
@@ -23,6 +23,8 @@ manifest:
       url-base: https://github.com/zephyrproject-rtos
     - name: babblesim
       url-base: https://github.com/BabbleSim
+    - name: zephyr-gb
+      url-base: https://github.com/devinjaenicke
```

Comment:
No need to do this, just reference the PR directly as instructed in https://github.com/zephyrproject-rtos/hal_silabs/pull/135#pullrequestreview-3199992217, i.e. you only need to update the `revision:` line.